src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyDeviceChallengePoll.ts

```diff
@@ -30,23 +31,38 @@ import {
 import { hasMinAuthenticatorOptions, loc, updateTransactionWithNextStep } from '../../../util';
 
 const getTitleText = (challengeMethod: string) => {
-  if (challengeMethod === CHALLENGE_METHOD.APP_LINK) {
-    return loc('appLink.title', 'login');
-  }
-  if (challengeMethod === CHALLENGE_METHOD.UNIVERSAL_LINK) {
-    return loc('universalLink.title', 'login');
+  switch (challengeMethod) {
+    case CHALLENGE_METHOD.APP_LINK:
+      return loc('appLink.title', 'login');
+
+    case CHALLENGE_METHOD.CHROME_DTC:
+      return loc('chrome_dtc.title', 'login');
+
+    case CHALLENGE_METHOD.CUSTOM_URI:
+      return loc('customUri.title', 'login');
+
+    case CHALLENGE_METHOD.UNIVERSAL_LINK:
+      return loc('universalLink.title', 'login');
+
+    default:
+      return '';
```

Comment:
It seems like (based on the logic before the update) uses `return loc('customUri.title', 'login');` as the default?

Reply:
yeah I remember I wrote that and made it look like custom uri is the default fallback case but thats not actually needed.  I looked at how gen 2 handles it and confirmed they use a switch with a specific custom uri case: https://github.com/okta/okta-signin-widget/blob/9d0a36dbe47c28a4df989da89b669e51d26a6996/src/v2/view-builder/utils/ChallengeViewUtil.js#L46

Reply:
Yeah we had a discussion about this above already @glenfannin-okta, but thanks for the callout and thanks for following up on this @leemchale-okta!